packages/azure-kusto-ingest/README.md

```diff
@@ -237,3 +236,30 @@ async function ingestFromFile() {
     await waitForStatus();
 }
 ```
+
+### Browser run
+
+The code is browser compatible with main differences in the run are that ingestFromFile accepts a Blob object and
+ingestFromStream accepts an ArrayBuffer. Fallback files are called '*.browser.ts'.
```

Comment:
The code is browser-compatible, with the main differences being that ingestFromFile accepts a Blob object and ingestFromStream accepts an ArrayBuffer. Fallback files are called '*.browser.ts'.


(also, mention changes in authentication?)

Reply:
This README is for the ingest library only